docs/hubs/expense/api-documentation.md

```diff
@@ -0,0 +1,18 @@
+---
+heading: Expense APIs
+seo: Expense API Docs | Cloud Elements API Docs
+title: Platform Documentation
+description: Expense Hub API Documentation.
+layout: sidebarapidocs
+apis: API Docs
+platform: expense
+breadcrumbs: /docs/hubs/hub-docs.html
+parent: Back to Hub Docs
+order: 90
+---
+
+{% include swagger.html %}
+
+{% include padding-bottom.html %}
+{% include padding-bottom.html %}
```

Comment:
Do you like doing the spacing at the bottom of the page this way, or would you rather have some site-wide default for this?